lab-taylor/lib/player-crud.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const debug = require('debug')('soccer:player-crud');
+const Player = require('../model/player');
+const teamCrud = require('../lib/team-crud');
+const AppError = require('./AppError');
+
+exports.createPlayer = function (reqBody) {
+  debug('creating player');
+  return new Promise((resolve, reject) => {
+    if (!reqBody.name) return reject(AppError.error400('players require a name'));
+    if (!reqBody.hometown) return reject(AppError.error400('players require a hometown'));
+    if (!reqBody.position) return reject(AppError.error400('players require a position'));
+    if (!reqBody.number) return reject(AppError.error400('players require a number'));
+    if (!reqBody.teamId) return reject(AppError.error400('players need to have a team'));
+
+    reqBody.timestamp = new Date();
+    const player = new Player(reqBody);
```

Comment:
Be careful using const on a variable like this. It works here, but if you're modifying it, you'll get an error.